docs/docs/examples/elasticsearch/BITBUCKET_ELASTICSEARCH.md

```diff
@@ -70,3 +79,7 @@ bitbucket:
       passwordSecretKey: password
 ```
 > Read about [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/){.external}.
```

Comment:
Can we replace all of these callouts (`>`) with an appropriate [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) in this case this would be an `!!!info```

Reply:
Think you forgot to do this one.